app/views/ingestibles/_form.html.haml

```diff
@@ -1,22 +1,23 @@
-= form_for @ingestible do |f|
```

Comment:
It is hard to use single form approach for ingestible edit page, as we need to update different entities from it.
So I've made every tab to have own form updated separately. This approach has some drawbacks though (see PR description)